### PR TITLE
Fix views to avoid double dots

### PR DESCRIPTION
**Pull request by shalapatil**
## Notion card
https://www.notion.so/saeloun/Refactor-complex-queries-95d24c159b794fa8a211fa4c9a427ea1

## Summary
Views were using double dots, so fixed it by passing required data from the controller. Moved queries to controller.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor (non-breaking and retains same functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [x] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [ ] I have annotated changes in the PR that are relevant to the reviewer
- [ ] I have added automated tests for my code

### CODE REVIEW

**Comment by shalapatil** [comment]
Cypress tests are failing

**Comment by shalapatil** [comment]
CI Test Suite was passing before merging latest develop,
Now there is no CI, buildkite is failing.

File: app/controllers/internal_api/v1/invoices_controller.rb [hunk 1]
```diff
@@ -31,7 +31,11 @@ def create
 
   def edit
     authorize invoice
-    render :edit, locals: { invoice: }
+    render :edit, locals: {
+      invoice:,
+      client: invoice.client,
+      current_company_clients: current_company.client_list
```
[the remark this answers is not in the document]
Comment (shalapatil) [reply app/controllers/internal_api/v1/invoices_controller.rb]: done

**Comment by shalapatil** [comment]
@apoorv1316 Please test following on staging
- Invoice list page
- Filtering on invoice page
- Invoice details page
- Update client of the invoice

**Comment by apoorv1316** [comment]
@shalapatil Getting an error on the edit invoice if I add a manual entry to it.  Please check the loom. cc @supriya3105 https://www.loom.com/share/4cf7019e863b4681a7cdbac586cda14e

**Comment by shalapatil** [comment]
Even this is working fine on my end 
Can we please confirm if staging is working fine as expected. 
Also, there is one other ticket open for handling manual invoice line entries

**Pull request merged**